content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -377,14 +377,12 @@ Here are some examples of device plugin implementations:
 * The [AMD GPU device plugin](https://github.com/RadeonOpenCompute/k8s-device-plugin)
 * The [Intel device plugins](https://github.com/intel/intel-device-plugins-for-kubernetes) for
   Intel GPU, FPGA, QAT, VPU, SGX, DSA, DLB and IAA devices
-* The [KubeVirt device plugins](https://github.com/kubevirt/kubernetes-device-plugins) for
-  hardware-assisted virtualization
 * The [NVIDIA GPU device plugin for Container-Optimized OS](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu)
-* The [RDMA device plugin](https://github.com/hustcat/k8s-rdma-device-plugin)
```

Comment:
Why should we remove this?

Reply:
So RDMA is supported with SR-IOV device plugin and the shared RDMA use-case with https://github.com/Mellanox/k8s-rdma-shared-dev-plugin. The project it self seem to maintained and I opened an issue if it maintained https://github.com/hustcat/k8s-rdma-device-plugin/issues/22 asking that question. 

Also I think it better to provide examples of device plugins that companies responsible from them then project from personal github.

Reply:
> I think it better to provide examples of device plugins that companies responsible from them then project from personal github.

We let our readers decide. Let's keep this one.